boards/riscv/visionfive2_jh7110/visionfive2_jh7110.dts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 Kanak Shilledar <kanakshilledar111@protonmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include "starfive/jh7110-visionfive-v2.dtsi"
+
+/ {
+        model = "StarFive VisionFive V2";
+        compatible = "starfive,visionfive-v2", "starfive,jh7110";
```

Comment:
Please move the `"starfive,jh7110"` compat to the SoC node.

Reply:
I have moved `model` and `compatible` properties from `boards/riscv/visionfive2/visionfive2_jh7110.dts` to `dts/riscv/starfive/jh7110-visionfive-v2.dtsi`

With reference from `hifive_unmatched` board devicetree.